sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -328,18 +328,19 @@ class SQLContext(@transient val sparkContext: SparkContext)
     def numPartitions = self.numShufflePartitions
 
     def strategies: Seq[Strategy] =
-      extraStrategies ++ (
```

Comment:
yeah, I just make this same as hive context way.
@chenghao-intel 
This test failed with `Refer to this link for build results (access rights to CI server needed): ` 
no rights to CL server ?